socorro/processor/processor_2015.py

```diff
@@ -85,37 +97,16 @@ class ProcessorRuleSets(RequiredConfig):
         required_config = Namespace()
 
         names = []
-        for (name, tag, rule_set_class_str, action_str, default_rules_str) in rule_sets:
+        for (name, default_rules_str) in rule_sets:
+            #name = "raw_to_processed_transform"
             names.append(name)
             required_config.namespace(name)
-            required_config[name].add_option(
-                name='tag',
-                doc='the lookup tag associated with this rule set',
-                default=tag
-            )
-            required_config[name].add_option(
-                name='rule_system_class',
-                default=rule_set_class_str,
-                doc='the fully qualified name of the rule system class',
-                from_string_converter=str_to_python_object,
-                likely_to_be_changed=True,
-            )
-            required_config[name].add_option(
-                name='action',
-                default=action_str,
-                doc=(
-                    'the name of the rule set method to run to processes '
-                    'these rules'
-                ),
-                likely_to_be_changed=True,
-            )
+
             required_config[name].add_option(
                 name='rules_list',
                 doc='a list of fully qualified class names for the rules',
                 default=default_rules_str,
-                from_string_converter=str_to_classes_in_namespaces_converter(
-                    name_of_class_option='rule_class'
-                ),
+                from_string_converter=str_to_classes_in_namespaces_converter(),
```

Comment:
After seeing the default rules str, I'm wondering how does this work?

Also, I have vague memories of you and I talking about taking rule sets out of configuration altogether, uplifting the configuration bits the rule classes needed to one of the processor classes, and passing that in when instantiating rule classes. Did that not work?